### PR TITLE
perlPackages.TextFormat: 0.61 -> 0.62

### PR DESCRIPTION
**Pull request by bcdarwin**
###### Motivation for this change

Routine update.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [NA] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/top-level/perl-packages.nix [hunk 1]
```diff
@@ -21230,10 +21230,10 @@ let
 
   TextFormat = buildPerlModule {
     pname = "Text-Format";
-    version = "0.61";
+    version = "0.62";
     src = fetchurl {
-      url = "mirror://cpan/authors/id/S/SH/SHLOMIF/Text-Format-0.61.tar.gz";
-      sha256 = "bb8a3b8ff515c85101baf553a769337f944a05cde81f111ae78aff416bf4ae2b";
+      url = "mirror://cpan/authors/id/S/SH/SHLOMIF/Text-Format-0.62.tar.gz";
```
Comment (SuperSandro2000) [inline pkgs/top-level/perl-packages.nix:21235]: If we add rec to buildPerlModule we could replace the version with a varible.

**Comment by stigtsp** [comment]
@GrahamcOfBorg build perlPackages.TextFormat

**Review by stigtsp — APPROVED** [review]
LGTM

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 105598` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 105598` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>5 packages built:</summary>
  <ul>
    <li>conglomerate</li>
    <li>minc_tools</li>
    <li>minc_widgets</li>
    <li>perl530Packages.TextFormat</li>
    <li>perl532Packages.TextFormat</li>
  </ul>
</details>

**Pull request merged**